notebooks/getting_started/part3_exploring_cohorts.ipynb

```diff
@@ -1,15 +1,5 @@
 {
```

Comment:
<p>I think you also need this line:</p><p> </p><p><code>study_UID = selection_df["StudyInstanceUID"].values[0]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ImagingDataCommons/IDC-Tutorials/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>This line is actually just a few cells down in the visualization section now.</p>